metaflow/plugins/kubernetes/kubernetes_job.py

```diff
@@ -135,6 +138,10 @@ def create(self):
                                         "METAFLOW_KUBERNETES_SERVICE_ACCOUNT_NAME": "spec.serviceAccountName",
                                         "METAFLOW_KUBERNETES_NODE_IP": "status.hostIP",
                                     }.items()
+                                ]
+                                + [
+                                    client.V1EnvVar(name=k, value=str(v))
+                                    for k, v in inject_tracing_vars({}).items()
```

Comment:
this is a bit of a headscratcher, as there are two implementations of this function, but the one imported here seems effectively a no-op?